_includes/archive.html

```diff
@@ -0,0 +1,24 @@
+{% assign currentDate = post.date | date: "%Y" %}
+{% if currentDate != myDate %}
+<h1 class="is-size-2">{{ currentDate }}</h1>
+{% assign myDate = currentDate %}
+{% endif %}
+<div class="card article">
+  <div class="card-content">
+    <div class="media">
+      <div class="media-content has-text-centered">
+        <p class="title article-title"><a href="{{post.url}}">{{ post.title}}</a></p>
+        <p class="subtitle is-6 article-subtitle">
+          <i class="fas fa-calendar-alt"></i> {{ post.date | date: "%-d %B, %Y"}} &nbsp;&nbsp;
+          {% for category in post.categories %}
+          <a href="/category/{{category}}">
```

Comment:
indentation

Reply:
I am not able to figure out ! Please provide the detail :)

Reply:
Indent the `a` tag inside for loop

Reply:
okay done for archive.html as well.